Dockerfile

```
@@ -0,0 +1,21 @@
+FROM node:12.6-alpine
```

```suggestion
FROM node:12.16-alpine
```

12.6は安定版ではないのでLTSに入ったバージョンのものを使うのが望ましいと思います。

---

デプロイスクリプトに10.x系と書いてあるので10.x系に合わせた方が良いかもしれません
(私の手元は12系ですが)

https://github.com/tokyo-metropolitan-gov/covid19/blob/staging/.github/workflows/develop.yml#L17

---

合わせておいた方が良さそうなので修正します！

---

@mikkame それでいうとpackage.jsonのengineとかで統一するのもありかもしれませんね